src/tools/dev/scripts/bv_support/bv_mili.sh

```diff
@@ -1067,7 +1067,7 @@ function build_mili
             warn "Mili build failed.  Giving up"
             return 1
         fi
-    elif [[ ${MILI_VERSION} == 13.1.1-patch || ${MILI_VERSION} == 15.1 ]] ; then
+    elif [[ ${MILI_VERSION} == 13.1.1-patch || ${MILI_VERSION} == 15.1 || ${MILI_VERSION} == 19.2 ]] ; then
```

Comment:
Do you really need to keep the older mili version logic here? Especially for 13.1.1-patch and 1.11.1 in code below?

Reply:
Good question. I'm actually not really sure why the older versions were left in the first place... Do you know of any reason to keep them around? I can't think of a situation where we'd want to use an older version of mili within a new release.

Reply:
I cannot think of a reason to keep the older version stuff around.
Perhaps @brugger1 or @cyrush can think of one?